### PR TITLE
Added 3 python questions

### PR DESCRIPTION
**Pull request by Suhas2002**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: src/data/python-quiz.ts [hunk 1]
```diff
@@ -1,4 +1,37 @@
 const pythonQuiz = [
+  {
+    Question: 
+      "Which of these can you use as a dictionary keys?",
```
Comment (jdwilkin4) [inline src/data/python-quiz.ts:4]:
```suggestion
      "In Python, which of following is the correct syntax for a key-value pair in a dictionary?",
```

File: src/data/python-quiz.ts [hunk 2]
```diff
@@ -1,4 +1,37 @@
...
+    Answer: "('key', 'value')",
+    Distractor1: "5j",
+    Distractor2: "'key'",
+    Distractor3: "'count'",
+    Explanation: 
+      "There is only option with brackets.",
```
Comment (jdwilkin4) [inline src/data/python-quiz.ts:10]:
```suggestion
      "A Python dictionary is a sequence of key-value pairs separated by commas. Here is the correct syntax: key1: value1.",
```

File: src/data/python-quiz.ts [hunk 3]
```diff
@@ -1,4 +1,37 @@
 const pythonQuiz = [
+  {
+    Question: 
+      "Which of these can you use as a dictionary keys?",
+    Answer: "('key', 'value')",
```
Comment (jdwilkin4) [inline src/data/python-quiz.ts:5]:
```suggestion
    Answer: " key1: value1",
```

File: src/data/python-quiz.ts [hunk 4]
```diff
@@ -1,4 +1,37 @@
...
+    Question: 
+      "What is the index of the first character in a string?",
+    Answer: "0",
+    Distractor1: "-1",
+    Distractor2: "1",
+    Distractor3: "A",
+    Explanation: 
+      "Strings are zero-indexed: The index of a string's first character is 0 , and the index of a string's last character is the length of the string minus 1",
+    Link: "https://www.freecodecamp.org/news/how-to-format-a-string-in-python/",
+  },
```
Comment (jdwilkin4) [inline src/data/python-quiz.ts:34]: This one is more a general programming question and Python specific.
Plus we already have similar questions in the quiz files.
Please remove this.

File: src/data/python-quiz.ts [hunk 5]
```diff
@@ -1,4 +1,37 @@
...
+    Distractor1: "5j",
+    Distractor2: "'key'",
+    Distractor3: "'count'",
+    Explanation: 
+      "There is only option with brackets.",
+    Link: "https://www.freecodecamp.org/news/python-dictionary-methods-dictionaries-in-python/",
+  },
+  {
+    Question: 
+      "What will the following Python code print? print(len('Hello world!'))",
```
Comment (jdwilkin4) [inline src/data/python-quiz.ts:15]:
```suggestion
      "In Python, what is the output for this code: print(len('Hello world!'))?",
```

File: src/data/python-quiz.ts [hunk 6]
```diff
@@ -1,4 +1,37 @@
...
+    Answer: "12",
+    Distractor1: "11",
+    Distractor2: "10",
+    Distractor3: "13",
+    Explanation: 
+      "Hello is 5 characters, world! is 6 characters and there is a space in between them. So, the length is 12",
```
Comment (jdwilkin4) [inline src/data/python-quiz.ts:21]:
```suggestion
      "The len function returns all characters within a string. Every single character(and space) in Hello world! would result in a length of 12.",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

File: src/data/python-quiz.ts (hunk 4 above)
Comment (Suhas2002) [reply src/data/python-quiz.ts:34]: Yeah sure

**Comment by Suhas2002** [comment]
Hey @jdwilkin4 
I have done all the changes you requested for

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**